app/controllers/admin/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Admin::ForgotPasswordsController < ApplicationController
+  before_action :load_admin, only: :create
+
+  def create
+    message = ForgotPasswordsService.call @admin
+    render json: message
+  end
+
+  private
+  def load_admin
+    @admin = Admin.find_by email: params[:email]
```

Comment:
em cho nó raise lỗi luôn.
```suggestion
    @admin = Admin.find_by! email: params[:email]
```

Reply:
rồi ném cho ErrorHandler nó render json